packages/ag-charts-community/src/api/preset/sparkline.ts

```diff
@@ -182,18 +186,21 @@ export function sparklineDataPreset(data: any[] | undefined): {
             const mappedData = data.map((y, x) => ({ x, y }));
             return { data: mappedData, series: [{ xKey: 'x', yKey: 'y' }] };
         } else if (Array.isArray(firstItem)) {
-            const mappedData = data.map(([x, y]) => ({ x, y }));
+            const mappedData = data.filter((item) => !!item).map(([x, y]) => ({ x, y }));
```

Comment:
We shouldn't be doing this filtering here - what is the motivation for this?

Reply:
Our current specs state that we ignore undefined entries in the array - happy to move the filtering elsewhere. Discovered it when one of our examples threw an NPE here

Reply:
Moving to the grid's sparklineRenderer